src/ngraph/runtime/interpreter/int_executable.cpp

```diff
@@ -63,6 +63,15 @@ runtime::interpreter::INTExecutable::INTExecutable(const shared_ptr<Function>& f
     : m_is_compiled{true}
     , m_performance_counters_enabled{enable_performance_collection}
 {
+    auto is_supported = [](const Node& node) {
```

Comment:
Why is this here? Also, it can be simplified to `is_type<op::v0::Clamp>`

Reply:
This allows the interpreter to run reference kernels for the fused ops as opposed to running the decomposed reference kernels.  Should I remove it or simplify it?